### PR TITLE
Add notes about lerna to README

### PR DESCRIPTION
**Pull request by LandryNorris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
+
+## package structure
+
+This project uses [lerna](https://lerna.js.org) for
+multi-package builds.
+
+We have several packages currently:
+
+1. rhsplib: Raw interaction with RHSPLib.
+1. expansion-hub: High-level interaction with a REV expansion hub.
```
Comment (LandryNorris) [inline README.md:13]: Markdown spec allows us to number all items '1.', and compliant viewers should show the proper numbers. This allows us to easily add an item to the middle of the list without having to worry about committing a large number of changes when later items have the number change.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,30 @@
...
+If you have other tasks you wish to create, define 
+them in the package.json like normal, and run 
+`lerna run mytask`
 
 ## Making a release
 
 1. Check out the `main` branch
 2. Update `version` field in `package.json`
-3. Run `npm install`
+3. Run `lerna run build`
```
Comment (LandryNorris) [inline README.md:27]: We should check if there are any changes we need to make to publish a lerna package

File: README.md (hunk 1 above)
Comment (NoahAndrews) [reply README.md:13]: Hmm, that's a clever way to abuse the spec. However, it hurts the readability of the plain text version, which I am not a fan of. I think I'd rather have them be the correct numbers, even if it means that more changes are needed when editing the list. A good markdown editor should let you auto-fix that.

File: README.md (hunk 2 above)
Comment (NoahAndrews) [reply README.md:27]: `npm install` is still necessary

File: README.md [hunk 3]
```diff
@@ -1,12 +1,30 @@
...
+1. rhsplib: Raw interaction with RHSPLib.
+1. expansion-hub: High-level interaction with a REV expansion hub.
+1. sample: Command line app useful for testing and showing features.
+
+## building and other tasks
+
+To build the app, run `lerna run build`.
+If you have other tasks you wish to create, define 
+them in the package.json like normal, and run 
+`lerna run mytask`
```
Comment (NoahAndrews) [inline README.md:21]: This is only necessary when you're defining the tasks in the per-package `package.json` files, right? Tasks defined in the top-level `package.json` file can be run using `npm run`.

**Review by NoahAndrews — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (LandryNorris) [reply README.md]: Added a note

File: README.md [hunk 4]
```diff
@@ -1,6 +1,28 @@
...
+## package structure
+
+This project uses [lerna](https://lerna.js.org) for
+multi-package builds.
+
+We have several packages currently:
+
+1. rhsplib: Raw interaction with RHSPLib.
+2. expansion-hub: High-level interaction with a REV expansion hub.
+3. sample: Command line app useful for testing and showing features.
```
Comment (NoahAndrews) [inline README.md:14]:
```suggestion
1. `rhsplib`: Raw interaction with RHSPLib.
2. `expansion-hub`: High-level interaction with a REV expansion hub.
3. `sample`: Command line app useful for testing and showing features.
```

File: README.md [hunk 5]
```diff
@@ -1,6 +1,28 @@
 # node-rhsplib
 
-This package enables Node.js applications to control devices that speak the REV Hub Serial Protocol (such as the REV Robotics Expansion Hub).
+This project enables Node.js applications to control devices that speak the REV Hub Serial Protocol (such as the REV Robotics Expansion Hub).
+
+## package structure
```
Comment (NoahAndrews) [inline README.md:5]:
```suggestion
## Package structure
```

File: README.md [hunk 6]
```diff
@@ -1,6 +1,28 @@
...
+This project uses [lerna](https://lerna.js.org) for
+multi-package builds.
+
+We have several packages currently:
+
+1. rhsplib: Raw interaction with RHSPLib.
+2. expansion-hub: High-level interaction with a REV expansion hub.
+3. sample: Command line app useful for testing and showing features.
+
+## building and other tasks
```
Comment (NoahAndrews) [inline README.md:16]:
```suggestion
## Building and other tasks
```

File: README.md [hunk 7]
```diff
@@ -1,6 +1,28 @@
...
+
+To build the app, run `lerna run build`.
+If you have other tasks you wish to create, define 
+them in the module's package.json like normal, and run 
+`lerna run mytask`. This will run 'mytask' in every
+module that has a task with that name.
+
+Tasks in the top-level package.json can be run with 
+npm run like normal.
```
Comment (NoahAndrews) [inline README.md:25]:
```suggestion
`npm run` like normal.
```

**Review by NoahAndrews — COMMENTED** [review]
Much better, thanks. I believe the section about releasing still needs updating, but for now we can just mark it as outdated.

**Comment by NoahAndrews** [comment]
> Much better, thanks. I believe the section about releasing still needs updating, but for now we can just mark it as outdated.

Please mark said section as outdated.

**Comment by LandryNorris** [comment]
Marked the release section as needing updating

**Review by NoahAndrews — APPROVED** [review]

**Pull request merged**